stack/logging.rst

```diff
@@ -93,24 +116,29 @@ For example:
 Log Levels
 ==========
 
-:lmod:`lsst.log` has six levels; in increasing order of severity the are: ``TRACE`` < ``DEBUG`` < ``INFO`` < ``WARN`` < ``ERROR`` < ``FATAL``.
+:mod:`logging` has five standard levels; in increasing order of severity the are: ``DEBUG`` < ``INFO`` < ``WARNING`` < ``ERROR`` < ``CRITICAL``.
 The guideline of using the log levels is as follows:
 
-- FATAL: for severe errors that may prevent further execution of the component.
+- CRITICAL: for severe errors that may prevent further execution of the component (FATAL is also allowed as an alias).
 - ERROR: for errors that may still allow the execution to continue.
-- WARN: for conditions that may indicate a problem but that allow continued execution.
+- WARNING: for conditions that may indicate a problem but that allow continued execution (WARN is also allowed as an alias).
 - INFO: for information that is of interest during normal execution including production.
 - DEBUG: for information that is of interest to developers but not of interest during production.
+
+In addition there are two additional log levels allowed for specialist pipelines-specific loggers (such as those used for :lclass:`lsst.pipe.base.Task`):
+
+- VERBOSE: for messages of a more detailed nature than would normally be expected to be shown by default but that will not swamp the user in the way that DEBUG messages would.
```

Comment:
I'd still like to say here "But consider using DEBUG messages with a separate logger name that can be enabled when desired, as described below."

Reply:
Do you mean the section below talking about `TRACEn.` logger names? Or do you mean the code immediately below here? ie are you advocating a `verbose.lsst.xxx` hierarchy reserved for verbose messages?

Reply:
Doesn't have to start with `verbose`.  That depends on whether synchronized verbosity across different modules is desirable or just increased verbosity in one place.